Sparkle/SUFileManager.h

```diff
@@ -50,7 +52,7 @@
  * When moving an item from a source to a destination, it is desirable to create a temporary intermediate destination on the same volume as the destination to ensure
  * that the item will be moved, and not copied, from the intermediate point to the final destination. This ensures file atomicity.
  */
-- (NSURL *)makeTemporaryDirectoryWithPreferredName:(NSString *)preferredName appropriateForDirectoryURL:(NSURL *)appropriateURL error:(NSError **)error;
+- (nullable NSURL *)makeTemporaryDirectoryWithPreferredName:(NSString *)preferredName appropriateForDirectoryURL:(nullable NSURL *)appropriateURL error:(NSError *_Nullable*_Nullable)error;
 
```

Comment:
To clarify, it's nullable in Apple's API because their multi-purpose method depending on what set of flags you pass is a little poorly designed.